Move spinal cord CSV files to `spinal_cord` folder

valosekj opened the pull request:
This PR moves CSV files with spinal cord morphometrics from the root repo directory to the newly created `spinal_cord` folder.

Related: https://github.com/spinalcordtoolbox/PAM50-normalized-metrics/issues/15

sandrinebedard reviewed (approved):
Looks good! Thanks @valosekj  for the changes! Maybe we could also specify in the readme that the csv files are under the `spine_cord` folder

valosekj commented:
Good point! I added info about the `spinal_cord` folder and refactored the README a little bit: https://github.com/spinalcordtoolbox/PAM50-normalized-metrics/blob/jv/15_add_folders_to_improve_repo_organization/README.md

What do you think?

sandrinebedard commented:
> Good point! I added info about the `spinal_cord` folder and refactored the README a little bit: https://github.com/spinalcordtoolbox/PAM50-normalized-metrics/blob/jv/15_add_folders_to_improve_repo_organization/README.md
> 
> What do you think?

Looks good!
